sceptre/logcentral/config/prod/ConfigLogAggregation.yaml

```diff
@@ -0,0 +1,7 @@
+template_path: ConfigLogAggregation.j2
+stack_name: ConfigLogAggregation
+sceptre_user_data:
+  stackname: !stack_name
+  config_bucket: !stack_output_external sagebase-cloudtrail::sagebaseDashcloudtrailDashCloudTrailBucket
```

Comment:
Is this bucket name correct? Is there a different `sagebase-clonfig::sagebaseDashconfigtrailDashConfigTrailBucket` output?

Reply:
good catch.  I've updated this to reference the master config bucket.  I also changed this PR to deploy with ofn instead of sceptre.